pkg/istio-agent/sds-agent.go

```diff
@@ -524,6 +524,7 @@ func applyEnvVars() {
 	workloadSdsCacheOptions.SecretTTL = secretTTLEnv
 	workloadSdsCacheOptions.SecretRotationGracePeriodRatio = secretRotationGracePeriodRatioEnv
 	workloadSdsCacheOptions.RotationInterval = secretRotationIntervalEnv
+	workloadSdsCacheOptions.TrustDomain = trustDomainEnv
```

Comment:
Just curious, do you know why this does not exist? Was it removed?

Reply:
I think the code was refactored recently.

Reply:
Isn't trust domain in ProxyConfig ?